astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -724,14 +722,13 @@ def __init__(
             parent=parent,
         )
 
-    def postinit(self, elt=None, generators=None):
+    def postinit(self, elt=None, generators: Optional["nodes.Comprehension"] = None):
         """Do some setup after initialisation.
 
         :param elt: The element that forms the output of the expression.
         :type elt: NodeNG or None
```

Comment:
```suggestion
    def postinit(self, elt: "nodes.NodeNG" =None, generators: Optional["nodes.Comprehension"] = None):
        """Do some setup after initialisation.

        :param elt: The element that forms the output of the expression.
```
Wondering also if it should be ``Optional[List["nodes.Comprehension"]]``

Reply:
Well, that was quite stupid... Fixed!